dev/devicelab/manifest.yaml

```diff
@@ -236,6 +236,7 @@ tasks:
     description: >
       Measures the startup time of the Complex Layout sample app on Android.
     stage: devicelab
+    flaky: true
```

Comment:
Why mark so many as flaky? Isn't this just changing the behavior of only the transition test for now?

Reply:
Although this test doesn't wait for the first frame, the way that we measure the `start_up` time changes. Hence I marked all `start_up` tests to be flaky.